File: src/main/java/no/nav/fo/veilarbregistrering/domain/ManuellRegistrering.java
package no.nav.fo.veilarbregistrering.domain;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
@ToString
@EqualsAndHashCode
public class ManuellRegistrering {

    long id;
    String aktorId;
    String veilederIdent;
    String veilederEnhetId;

}

File: src/main/java/no/nav/fo/veilarbregistrering/registrering/manuell/ManuellRegistrering.kt
package no.nav.fo.veilarbregistrering.registrering.manuell

import no.nav.fo.veilarbregistrering.registrering.BrukerRegistreringType

data class ManuellRegistrering (
    val id: Long = -1,
    val registreringId: Long,
    val brukerRegistreringType: BrukerRegistreringType,
    val veilederIdent: String,
    val veilederEnhetId: String,
){
    constructor(
        registreringId: Long,
        brukerRegistreringType: BrukerRegistreringType,
        veilederIdent: String,
        enhetsId: String
    ) :
            this(
                registreringId = registreringId,
                brukerRegistreringType = brukerRegistreringType,
                veilederIdent = veilederIdent,
                veilederEnhetId = enhetsId,
            )

    override fun toString(): String {
        return "ManuellRegistrering(id=$id, registreringId=$registreringId, brukerRegistreringType=$brukerRegistreringType, veilederIdent=$veilederIdent, veilederEnhetId=$veilederEnhetId)"
    }
}